currencies/ticker_task_test.go

```diff
@@ -0,0 +1,306 @@
+package currencies_test
+
+import (
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/prebid/prebid-server/currencies"
+	"github.com/prebid/prebid-server/util/timeutil"
+	"github.com/stretchr/testify/assert"
+)
+
+type MockRunner struct {
+	updateNotifier chan<- int
+	runCount       int
+}
+
+func (mcc *MockRunner) Run() error { return nil }
+func (mcc *MockRunner) Notify()    { mcc.runCount++; mcc.updateNotifier <- mcc.runCount }
```

Comment:
Let's remove `Notify` and have the `Run` method increase the count for the test to inspect.